### PR TITLE
NOISSUE - Use log level error for VermeMQ docker

### PR DESCRIPTION
**Pull request by manuio**
Signed-off-by: Manuel Imperiale <manuel.imperiale@gmail.com>

### CODE REVIEW

**Review by drasko — DISMISSED** [review]
LGTM

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -208,8 +208,8 @@ services:
     container_name: mainflux-vernemq
     restart: on-failure
     environment:
-      DOCKER_VERNEMQ_ALLOW_ANONYMOUS: "on"
-      DOCKER_VERNEMQ_LOG__CONSOLE__LEVEL: error
+      DOCKER_VERNEMQ_ALLOW_ANONYMOUS: ${MF_DOCKER_VERNEMQ_ALLOW_ANONYMOUS}
```
Comment (drasko) [inline docker/docker-compose.yml:211]: Would ommiting `""` here create problem? I guess not, just asking if this was tested?

Comment (manuio) [reply docker/docker-compose.yml:211]: AFAIK for docker-compose it's the same. But I didn't deeply tested.

Comment (nmarcetic) [reply docker/docker-compose.yml:211]: This will never be `off` because Manflux doesn't use Broker authentication. IMHO no need for env var.

**Review by drasko — APPROVED** [review]
LGTM

**Review by mteodor — APPROVED** [review]

**Review by blokovi — APPROVED** [review]

**Pull request merged**